build/asset-manifest.json

```diff
@@ -0,0 +1,16 @@
+{
+  "files": {
+    "main.css": "/static/css/main.b1502b86.chunk.css",
+    "main.js": "/static/js/main.2f1b9c41.chunk.js",
+    "main.js.map": "/static/js/main.2f1b9c41.chunk.js.map",
+    "runtime~main.js": "/static/js/runtime~main.a8a9905a.js",
+    "runtime~main.js.map": "/static/js/runtime~main.a8a9905a.js.map",
+    "static/js/2.0bfae11e.chunk.js": "/static/js/2.0bfae11e.chunk.js",
+    "static/js/2.0bfae11e.chunk.js.map": "/static/js/2.0bfae11e.chunk.js.map",
+    "index.html": "/index.html",
+    "precache-manifest.77b12caf66a11380f362c2040fb2f6e4.js": "/precache-manifest.77b12caf66a11380f362c2040fb2f6e4.js",
+    "service-worker.js": "/service-worker.js",
+    "static/css/main.b1502b86.chunk.css.map": "/static/css/main.b1502b86.chunk.css.map",
+    "static/media/index.css": "/static/media/streaks-stripes-lines.6ea81463.png"
+  }
+}
```

Comment:
Add ./build folder to the .gitignore file
And remove this folder from the repository